app/src/app/[lng]/[inventory]/InventoryResultTab/TopEmissionsWidget.tsx

```diff
@@ -38,23 +39,28 @@ const TitleAndSelector = ({ t }: { t: Function }) => {
 };
 
 
-const EmissionsTable = ({ topEmissions }: { topEmissions: TopEmission[] }) => {
+const EmissionsTable = ({ topEmissions, t }: { topEmissions: TopEmission[], t: TFunction }) => {
   return (
     <TableContainer my={4}>
       <Table variant="simple">
         <Thead>
           <Tr>
-            <Th sx={{ "font": "bold", color: "black" }}>Subsector</Th>
-            <Th sx={{ "font": "bold", color: "black" }}>Total Emissions (CO2eq)</Th>
-            <Th sx={{ "font": "bold", color: "black" }}>% of Emissions</Th>
+            <Th sx={{ "font": "bold", color: "black" }}>{t("subsector")}</Th>
+            <Th sx={{ "font": "bold", color: "black" }}>{t("total-emissions-CO2eq")}</Th>
+            <Th sx={{ "font": "bold", color: "black" }}>{t("%-of-emissions")}</Th>
           </Tr>
         </Thead>
         <Tbody>
           {(topEmissions || []).map((emission, index) => (
             <Tr key={index}>
               <Td>
-                <Text fontSize="xl">{emission.subsectorName}</Text>
-                <Text fontSize="sm" color={"gray"}>{emission.sectorName}</Text>
+                <Text
+                  className="font-['Poppins'] text-sm leading-5 tracking-[0.5px]"
+                >{emission.subsectorName}</Text>
+                <Text
+                  color="#4B4C63"
```

Comment:
Use theme colors from app-theme.ts. They follow the same naming convention as the colors in Figma (but with a dot in the middle, like content.tertiary). They can be used instead of a hex color literal anywhere in Chakra UI.